modules/table/src/test/java/org/apache/ignite/internal/table/TxLocalCleanupRecoveryTest.java

```diff
@@ -67,4 +68,35 @@ public void testDeleteUpsertCommit() throws TransactionException {
 
         assertThrows(TransactionException.class, () -> deleteUpsert().commit());
     }
+
+    @Disabled("IGNITE-20560")
+    @Test
+    @Override
+    public void testTransactionAlreadyRolledback() {
+        // The value of 6 is higher than the default retry count.
+        // So we should give up retrying and crash.
+        failureCounter = new AtomicInteger(6);
```

Comment:
Could we somehow produce this number from the `default retry count` instead of using raw number 6?